### ISSUE TITLE
Visualizing the training using TensorBoard

### ISSUE THREAD

**Issue by smao-astro** [lululxvi/deepxde#77]
Hi Lu,

As far as I know the program prints the loss and metrics to standard output. I am wondering that do you know how should I modify the code so that I can save the training loss for every, for example, 500 epochs, and visualize the loss curve using TensorBoard? Thanks.

**Comment by lululxvi** [comment]
You can use `model.train(..., display_every=500)`. If you want to use TensorBoard, then you have to modify the source code model.py.
